p4-16/spec/P4-16-spec.mdk

```diff
@@ -4156,9 +4156,9 @@ of the header.
 "false". It can only be applied to an l-value.
 
 Similar to a `struct`, a header object can be initialized with a list
-expression [#sec-list-exprs] --- the list fields are assigned to the
+expression (see section [#sec-list-exprs]) --- the list fields are assigned to the
```

Comment:
Isn't "Section" supposed to be capitalized as well?

Reply:
Fine with me.

Reply:
https://github.com/p4lang/p4-spec/pull/1081/commits/43f0d41e6a791c682ff802771f731346ca770cc7